src/pkg/utils/io.go

```diff
@@ -59,8 +59,13 @@ type TextTemplate struct {
 }
 
 // MakeTempDir creates a temp directory with the zarf- prefix.
-func MakeTempDir() (string, error) {
-	tmp, err := os.MkdirTemp("", tmpPathPrefix)
+func MakeTempDir(basePath string) (string, error) {
+	if basePath != "" && InvalidPath(basePath) {
+		if err := CreateDirectory(basePath, 0700); err != nil {
+			return "", err
+		}
+	}
+	tmp, err := os.MkdirTemp(basePath, tmpPathPrefix)
 	message.Debugf("Creating temp path: '%s'", tmp)
```

Comment:
```suggestion
	message.Debugf("Created temp directory: %q", tmp)
```

Reply:
Didn't exactly do this but normalized the verbiage across things - noticed we were double printing the path in some cases as well.